### PR TITLE
[Docker] Enable docker log rotate when deploying and adding new node.

### PR DESCRIPTION
**Pull request by ydye**

### CODE REVIEW

File: pai-management/k8sPaiLibrary/maintaintool/docker-ce-install.sh [hunk 1]
```diff
@@ -17,6 +17,21 @@
...
+if command -v python >/dev/null 2>&1; then
+    echo python has been installed. And skip python install.
+else
+    apt-get update
+    apt-get -y install python
+    if command -v python >/dev/null 2>&1; then
+        echo Successfully install python
+    else
+        echo Failed install python
+        exit 1
```
Comment (fanyangCS) [inline pai-management/k8sPaiLibrary/maintaintool/docker-ce-install.sh]: >1 [](start = 13, length = 1)

please sync with Bin about this PR. I remember he asked to refine exit code.

Comment (ydye) [reply pai-management/k8sPaiLibrary/maintaintool/docker-ce-install.sh]: Sync with Hao Yuan, we only need to care about zore or nonzero. @hao1939

Comment (hao1939) [reply pai-management/k8sPaiLibrary/maintaintool/docker-ce-install.sh]: For now, a success/failure signal would be a great improvement.

File: pai-management/k8sPaiLibrary/maintaintool/docker-config-update.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
```
Comment (sterowang) [inline pai-management/k8sPaiLibrary/maintaintool/docker-config-update.py]: Actually this is a generic json update tool, suggest you make name more generic, and move it to a more common place.

Comment (ydye) [reply pai-management/k8sPaiLibrary/maintaintool/docker-config-update.py]: I don't want to expose this tool to others. And in code structure, I only hope this tool could be used when deploying or manage k8s or docker.  And this tool can only handle following json format.
```
{
key: value
key: value
}
``` 

And unable to handle the next json format.

```
[ "Ford", "BMW", "Fiat", {"a": "b"} ]
```

Comment (ydye) [reply pai-management/k8sPaiLibrary/maintaintool/docker-config-update.py]: I think in this PR, the most concerned problem is whether we should force enable log rotation of docker for user. Now, I force enable it when deploy or add a new machine, no matter the docker is installed by pai or user. @sterowang

Comment (sterowang) [reply pai-management/k8sPaiLibrary/maintaintool/docker-config-update.py]: Sure.

---
In reply to: [206409837](https://github.com/Microsoft/pai/pull/995#discussion_r206409837) [](ancestors = 206409837)

**Pull request merged**